dts/bindings/regulator/x-powers,axp2101-regulator.yaml

```diff
@@ -0,0 +1,139 @@
+# Copyright (c), 2021 NXP
+# Copyright (c), 2024 Lothar Felten <lothar.felten@gmail.com>
+# Copyright (c), 2024 Deng Baoan <dengbaoan@foxmail.com>
+# SPDX -License-Identifier: Apache-2.0
+
+description: |
+  AXP2101 PMIC
+
+  The PMIC has five DCDC converters and nine LDOs.
+  All need to be defined as children nodes.
+  For example:
+
+  i2c {
+    axp2101: axp2101@34 {
+      compatible = "x-powers,axp2101";
+      reg = <0x34>;
+      status = "okay";
+      axp2101_regulator:axp2101_regulator {
+        compatible = "x-powers,axp2101_regulator";
+        status = "okay";
+
+        DCDC1: DCDC1 {
```

Comment:
nodelabels not needed, status okay not needed, compatible wrong, etc.

Reply:
Reverting to old version without details.
The "enum: [0, 1]" is used by silabs,series2-dcdc.yaml. I assume that is ok to use.